## Summary Statistics

------

In [2]:
# Import the libraries
import pandas as pd
from pathlib import Path
import os
import matplotlib.pyplot as plt

In [4]:
# Import the original data files
df = pd.read_csv('../data/2017_diabetes_cleaned.csv')

--------

#### General Condition

In [5]:
df

diabete    bmi  physhlth  menthlth  michd  hchol  hblpr    vpa  grenda  \
0           1  39.94       0.0       0.0      0      0      1  105.0    10.0   
1           1  31.24       2.0       1.0      0      1      1    0.0    14.0   
2           0  24.33       0.0       0.0      0      0      1  112.0    17.0   
3           0  21.38       0.0       0.0      0      0      0    0.0     0.0   
4           0  28.97       0.0       0.0      0      1      1   15.0    10.0   
...       ...    ...       ...       ...    ...    ...    ...    ...     ...   
5641        1  28.34      30.0      30.0      1      1      1    0.0     0.0   
5642        0  28.70       0.0       0.0      0      0      0    0.0     0.0   
5643        0  26.63       0.0       0.0      0      0      0    0.0     0.0   
5644        0  28.19       4.0      30.0      0      0      0  120.0    13.0   
5645        0  24.21       0.0       0.0      0      1      1    0.0     7.0   

      potada  frnchda  frutda  hvdr  smok  incom50  cllgatt  sex  age65  
0        3.0      7.0    10.0     0     0        0        2    1      0  
1        0.0      2.0     2.0     0     0        0        3    1      0  
2        0.0      3.0     0.0     0     0        0        3    0      1  
3        3.0      0.0     0.0     0     0        0        2    0      1  
4       13.0      3.0    17.0     0     0        0        3    1      1  
...      ...      ...     ...   ...   ...      ...      ...  ...    ...  
5641     0.0      3.0     0.0     1     0        0        4    1      0  
5642    20.0      7.0     7.0     0     0        0        3    0      0  
5643     3.0      7.0    14.0     0     0        0        4    0      0  
5644    17.0      7.0    13.0     0     0        0        4    1      0  
5645     7.0      7.0    13.0     0     0        0        1    0      1  

[5646 rows x 18 columns]

##### After variable selection and data cleaning, we have got 5646 complete and clean samples in the dataset, with 18 variables.